src/main.cpp

```diff
@@ -1011,6 +1013,42 @@ int main(int argc, char **args)
   return 0;
 }
 
+void vcExtractAttributionText(vdkProjectNode *pNode, const char **ppCurrentText)
```

Comment:
This function should be a scene item function; its not only PointClouds that need this functionality long term